Jenkinsfiles/IT-release-on-demand-packages.groovy

```diff
@@ -45,18 +81,25 @@ timestamps {
                     grep uploaded release.ini"""
                 }
             }
-            checkpoint 'prepared'
-            /*
-            input 'Click ok to Perform Marketplace Packages'
-            stage('perform') {
-                dir('nuxeo/marketplace') {
-                    sh """#!/bin/bash -ex
-                    ../scripts/release_mp.py perform
-                    grep -C5 'Fail' release.ini || true
-                    grep uploaded release.ini
-                    """
+        }
+        checkpoint 'prepared'
+        stage('perform') {
+            def doPerform = input message: 'Do you want to Perform Marketplace Packages ?', parameters: [
```

Comment:
Rephrase:
> Do you want to perform the Nuxeo Packages release?

The two above greps are useful here, to decide whether to continue or not. Is it readable if printed here? Or is there a useful link to point at?

Reply:
Could you precise the issue or suggest a rephrase ?
The greps are visible on the build console:
```+ grep -C5 'skip = Failed' release.ini
+ true
+ grep uploaded release.ini
uploaded = https://nos-test-connect.nos.nuxeo.com/nuxeo: /opt/jenkins/workspace/Deploy/IT-release-on-demand-packages/nuxeo/marketplace/nuxeo-lambda-pictureview-conversion-marketplace/nuxeo-lambda-marketplace-package/target/nuxeo-lambda-marketplace-package-9.3-I20171019_0125.zip```